src/plugin/plg_starter_tunnel/index.go

```diff
@@ -12,8 +12,8 @@ package plg_starter_tunnel
 import (
 	"context"
 	"github.com/gorilla/mux"
-	. "github.com/mickael-kerjean/filestash/server/common"
-	"github.com/mickael-kerjean/remotedialer"
+	. "github.com/mickael-kerjean/filestash/src/common"
+	"github.com/BobCashStory/remotedialer"
```

Comment:
I'd rather keep my own dependency here again

Reply:
since remote dialer use file from filestash in his code i suggest to make a sub repo, i had to fork it to just use only my fork .